File: src/go/fib.go
package main

import (
	"fmt"
)

func main() {
  fmt.Printf("%d\n", fib(100))
}

func fib(n int) int {
  if n < 2 {
    return 1
  }
  return fib(n-1) + fib(n-2)
}

File: src/java/Fib.java
class Fib {
  public static void main(String[] args) {
    System.out.println(fib(100));
  }

  public static long fib(long n) {
    if (n < 2L) {
      return 1L;
    }
    return fib(n-1) + fib(n-2);
  } 
}
